### ISSUE TITLE
Fix travis CI

### ISSUE THREAD

**Issue by oscar-lima** [KCL-Planning/rosplan_demos#4]
```
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
rosplan_interface_movebase: Cannot locate rosdep definition for [rosplan_planning_system]
rosplan_interface_mapping: Cannot locate rosdep definition for [occupancy_grid_utils]
```

see: https://travis-ci.com/KCL-Planning/rosplan_demos/builds/110569351

**Comment by gerardcanal** [comment]
Fixed in #6. Thew travis config file was missing rosplan installation, needed to compile the demos.

There's a thing though, now is that this repo downloads ROSPlan from master, so if ROSPlan doesn't compile this repo won't either. We could fix a commit or tag in this travis.yml, but then I guess it would need to be kept updated. ROSPlan master should never fail if we don't accept failing PR, so it should be fine.

**Comment by oscar-lima** [comment]
Great! Thanks Gerard
